kolibri/core/device/task_notifications.py

```diff
@@ -0,0 +1,14 @@
+"""
+Tiny module to store strings for task notifications for device tasks.
+TODO: This can be migrated into kolibri/core/device/tasks.py in 0.17
```

Comment:
So I guess the underlying meaning of this is that we can't move this until 0.17 because we'll lose translations?